cli/command/cli.go

```diff
@@ -27,26 +27,28 @@ import (
 
 // Streams is an interface which exposes the standard input and output streams
 type Streams interface {
-	In() *InStream
-	Out() *OutStream
+	In() InputStreamer
+	Out() OutputStreamer
 	Err() io.Writer
 }
 
 // Cli represents the docker command line client.
 type Cli interface {
 	Client() client.APIClient
-	Out() *OutStream
+	Out() OutputStreamer
```

Comment:
I did this because I need to stub the method `IsTerminal()`, and I don't see any other way to do it if the Cli interface only accepts the specific `OutStream` struct. Do you see any other alternative?